docs/design-notes/cql-extensions-internal.md

```diff
@@ -0,0 +1,38 @@
+# Scylla CQL extensions
+
+Scylla extends the CQL language to provide a few extra features. This document
+lists those extensions.
+
+## SCYLLA_TIMEUUID_LIST_INDEX function
```

Comment:
Thank you for documenting other existing functions, that's going above and beyond.  Your extra effort improves everyone's life. :)